### PR TITLE
ZNC-26: Add Github Issue template 

### PR DESCRIPTION
**Pull request by LaureVergeron**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Use case
+
+Please provide use cases for changing the current behavior
+
+### Additional information
+
+- Is this request for your company? Y/N
+
+..- If Y: Company name:
+..- Are you using any Scality Enterprise Edition products (RING, Zenko EE)? Y/N
```
Comment (anurag4DSB) [inline .github/ISSUE_TEMPLATE.md]: Everything looks good to me. Not sure if "..-" was meant to be a bullet point.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- search the issue list for this repository (use the search bar, select
+  "Issues" on the left pane after searching);
+- if there is a duplicate, please do not open your issue, and add a comment
+  to the existing issue instead.
+
+--------------------------------------------------------------------------------
+
+## Bug report information
+
+(delete this section (everything between the lines) if you're not repoting a bug
```
Comment (JianqinWang) [inline .github/ISSUE_TEMPLATE.md]: typo 'reporting'

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (LaureVergeron) [reply .github/ISSUE_TEMPLATE.md]: Yes, it is, I used the wrong markup synta, good catch! fixed

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (LaureVergeron) [reply .github/ISSUE_TEMPLATE.md]: fixed, thanks!

**Comment by LaureVergeron** [comment]
@all: thanks for reviewing, adressed all your points. I have similar ones on Cloudserver and Backbeat, if you have the time to review :)

**Pull request merged**